### PR TITLE
test againts multiple eslint versions

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,10 +62,10 @@
     "semver": "7.3.2"
   },
   "peerDependencies": {
-    "eslint": "^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0"
+    "eslint": "^5.0.0 || ^6.0.0 || ^7.0.0"
```
Comment (ljharb) [inline package.json:65]: a breaking change isn’t needed here, I’d suggest reverting this line.

File: package.json [hunk 2]
```diff
@@ -44,6 +44,8 @@
     "@babel/preset-flow": "^7.9.0",
     "cross-env": "^7.0.2",
     "eslint": "^7.0.0",
+    "eslint-5": "npm:eslint@5",
```
Comment (ljharb) [inline package.json:47]: This is also not necessary. Make the dev dep line identical to the peer dep one, and explicitly install whichever version of eslint you want to test in CI.

Comment (amilajack) [reply package.json:47]: What are the tradeoffs between these two approaches?

Comment (ljharb) [reply package.json:47]: It’s less complicated, and works on npm versions that don’t understand the new and rarely used `npm:` protocol.

Also, having 3 eslints installed at once won’t actually be replicating what the user’s node_modules will be like, so the tests are less useful.

**Pull request merged**